server/src/test/java/org/opensearch/index/engine/InternalEngineTests.java

```diff
@@ -759,20 +761,20 @@ public long getProcessedCheckpoint() {
     }
 
     public void testFlushIsDisabledDuringTranslogRecovery() throws IOException {
-        engine.ensureCanFlush(); // recovered already
+        engine.translogManager().ensureCanFlush(); // recovered already
```

Comment:
This this indirection to translogManager() temporary and planned to be cleaned up in the follow on PRs?